weather.py

```diff
@@ -57,7 +85,32 @@ def load_data_from_csv(csv_file):
     Returns:
         A list of lists, where each sublist is a (non-empty) line in the csv file.
     """
-    pass
+    information_list = []
+
+    
+
+    print(csv_file)
+    with open(csv_file) as this_file:
+        my_reader = csv.reader(this_file) # take the information from the file
+        next(my_reader) # This command is going to skip one line
+
+        for line in my_reader:
+            if line == []:
+                continue
+            date,min,max = line 
```

Comment:
Nice work applying the more advanced concepts of `continue` and tuple unpacking!